app/gui/src/dashboard/hooks/backendHooks.ts

```diff
@@ -48,6 +48,7 @@ import {
 import type { MergeValuesOfObjectUnion } from 'enso-common/src/utilities/data/object'
 import { useMemo } from 'react'
 import { z } from 'zod'
+import { useMutationCallback } from '../utilities/tanstackQuery'
```

Comment:
would prefer `'#/` here - i know this is probably auto-imported but imo `#/utilities` isn't in the same 'slice' as `#/hooks`